namespaces/mson-namespace.md

```diff
@@ -85,22 +85,28 @@ Note: In MSON DOM _Nested Member Types_ _Type Section_ is the `content` of the e
 - `attributes`
     - `typeAttributes` (array) - _Type Definition_ attributes list, see _Type Attribute_  
         - (enum[string])
-            - required
-            - optional
-            - fixed
-            - sample
-            - default
-    - `variable` (boolean) - Element content is _Variable Value_
-    - `sample` - Alternative sample value for _Member Types_
+            - required - This element is required in parent's content
+            - optional - This element is optional in parent's content
+            - fixed - The `content` value is immutable.
+            - sample - The `content` value is a sample value.
+            - default - The `content` value is a default value.
 
-		The type of of `sample` attribute MUST match the type of element's `content`.
+    - `variable` (boolean) - The `content` value is a _Variable Value_
 
-    - `default` - Default value for _Member Types_
+      The `content` value is either a _Variable Type Name_, _Variable Value_ or _Variable Property Name_.
```

Comment:
You may need a little clarity here. Does this mean:

* If this is true, "The `content` value is either a _Variable Type Name_, _Variable Value_ or _Variable Property Name_."
* "The `content` value is either a _Variable Type Name_, _Variable Value_ or _Variable Property Name_," and if this is true, the content value is _Variable Type Name_

I think you can read it both ways here unless you put it together with the description above it.